strawberry/arguments.py

```diff
@@ -26,10 +39,31 @@ def get_arguments_from_annotations(
             origin_name=name,
             name=to_camel_case(name),
             origin=origin,
-            type=annotation,
             default_value=default_value,
         )
 
+        if get_origin(annotation) is Annotated:
+            annotated_args = get_args(annotation)
+
+            # The first argument to Annotated is always the underlying type
+            argument_definition.type = annotated_args[0]
+
+            argument_metadata = None
+            # Find any instances of StrawberryArgument in the other Annotated args,
+            # raising an exception if there are multiple StrawberryArguments
+            for arg in annotated_args[1:]:
+                if isinstance(arg, StrawberryArgument):
+                    if argument_metadata is not None:
+                        raise MultipleStrawberryArgumentsError(
+                            field_name=origin.__name__, argument_name=name
+                        )
+                    argument_metadata = arg
```

Comment:
this prevents from doing Annotated[str, argument(), argument()], right?

Reply:
Yep that's right